src/__tests__/reporter/retriesReporting.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('retries reporting', () => {
   test('client.startTestItem should be called with retry=true params', () => {
     const parentId = 'tempTestItemId';
     const expectedTestObj: StartTestObjType = {
-      startTime: reporter.client.helpers.now(),
+      startTime: expect.any(Number),
```

Comment:
What is the purpose of this change?

Reply:
It was part of my experiments, I'll revert it.